hoot-js/src/main/cpp/hoot/js/conflate/merging/ElementMergerJs.cpp

```diff
@@ -179,11 +177,13 @@ void ElementMergerJs::_merge(OsmMapPtr map, Isolate* current)
   // By convention, we're setting the status of any element that gets merged with something to
   // conflated, even if its yet to be reviewed against something else.
   ElementPtr mergedElement = map->getElement(mergeTargetId);
-  assert(mergedElement);
-  mergedElement->setStatus(Status(Status::Conflated));
-  mergedElement->getTags()[MetadataTags::HootStatus()] = "3";
-  mergedElement->getTags().remove(MetadataTags::HootMergeTarget());
-  LOG_VART(mergedElement);
+  if (mergedElement)
+  {
+    mergedElement->setStatus(Status(Status::Conflated));
+    mergedElement->getTags()[MetadataTags::HootStatus()] = "3";
+    mergedElement->getTags().remove(MetadataTags::HootMergeTarget());
+    LOG_VART(mergedElement);
+  }
```

Comment:
This is the part that fixes the assertion crash.

Reply:
So if the positive id is re-used and mergedElement is null, we still want to perform the three operations on whatever element gets returned, otherwise the `hoot:status` is missing and the `hoot:merge:target` is still present.

Reply:
The code updates now return the correct element ID that remains so that `mergedElement` contains the element with the `HootMergeTarget` tag.  That tag is then removed and the status updated to `conflated`.